lms/static/js/utils/navigation.js

```diff
@@ -0,0 +1,69 @@
+var edx = edx || {},
+
+    Navigation = (function() {
+
+        var navigation = {
+
+            init: function() {
+                if ($('.accordion').length) {
+                    navigation.openAccordion();
+                }
+            },
+
+            openAccordion: function() {
+                navigation.checkForCurrent();
+                navigation.listenForClick();
+            },
+
+            checkForCurrent: function() {
+                var active = $('.accordion .chapter-content-container:has(.active)').index('.accordion .chapter-content-container') ? $('.accordion .chapter-content-container:has(.active)').index('.accordion .chapter-content-container') : 0,
+                    activeSection = $('.accordion .button-chapter:eq(' + active + ')');
+
+                navigation.closeAccordions();
+                navigation.openAccordionSection(activeSection);
```

Comment:
This should only be called if there is an activeSection, so you probably don't need the activeSection variable and can define the element in the openAccordionSection call if active index > 0.